packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -90,11 +73,19 @@ describe('ConsoleLogExporter', () => {
       const exporter = new ConsoleExporter(defaultBufferConfig);
       const rootSpan = new RootSpan(tracer);
       rootSpan.startChildSpan('name', 'type', rootSpan.traceId);
-      const queue: RootSpan[] = [];
-      queue.push(rootSpan);
+      const queue: RootSpan[] = [rootSpan];
 
-      exporter.publish(queue);
-      assert.ok(true);
+      exporter.publish(queue)
+          .then((result: string) => {
+            assert.ok(result.indexOf(rootSpan.traceId) >= 0);
+            assert.ok(result.indexOf(rootSpan.id) >= 0);
+            assert.ok(result.indexOf(rootSpan.name) >= 0);
+            assert.ok(result.indexOf(rootSpan.spans[0].name) >= 0);
+            assert.ok(result.indexOf(rootSpan.spans[0].id) >= 0);
+          })
+          .catch((err: Error) => {
```

Comment:
I think I have the same comment as earlier, `.catch` is not needed if we just return `exporter.publish(queue).then()`. (BTW, without that change this test will pass synchronously before the asserts are run.)